pkg/controller/flink/flink.go

```diff
@@ -388,7 +391,8 @@ func (f *Controller) DeleteOldResourcesForApp(ctx context.Context, app *v1alpha1
 	}
 
 	for k := range deletedHashes {
-		f.LogEvent(ctx, app, corev1.EventTypeNormal, fmt.Sprintf("Deleted old cluster with hash %s", k))
+		f.LogEvent(ctx, app, corev1.EventTypeNormal, "ToreDownCluster",
```

Comment:
Same. Nit, Move reason to constant variable ?

Reply:
I don't really see any reason to. It's only used here and is descriptive of this event.

Reply:
Nothing major. Just a nit, so that they can be reused later.